src/README.md

```diff
@@ -1 +1,8 @@
 # Introduction
+
+To generate the book locally:
+
+```
+# Run in breez-sdk-docs:
+mdbook serve --open
+```
```

Comment:
There's a reference how to run it locally in the readme in the main folder. I think maybe this readme here can be deleted?

Reply:
Good point, done